server.py

```diff
@@ -1,41 +1,77 @@
 import subprocess
 
 from mcp.server.fastmcp import FastMCP
+from mcp.types import ToolAnnotations
 
 mcp = FastMCP("My App")
 
-@mcp.resource("config://app")
-def get_config() -> str:
-    """Static configuration data"""
-    return "App configuration here"
 
 @mcp.prompt()
-def review_code(code: str) -> str:
-    return f"Please review this code:\n\n{code}"
+def explain_code(code: str, level: str = "beginner") -> str:
+    """Explain code at different levels of detail"""
+    levels = {
+        "beginner": "Please explain this code in simple terms for someone new to programming",
+        "intermediate": "Please explain this code with moderate technical detail",
+        "advanced": "Please provide a detailed technical analysis of this code",
+    }
+    explanation_request = levels.get(level, levels["beginner"])
+    return f"{explanation_request}:\n\n```python\n{code}\n```"
 
-@mcp.tool()
-async def add_list_of_numbers(numbers: list[int]) -> int:
-    """Add a list of numbers"""
-    return sum(numbers)
 
-@mcp.tool()
-def count_letter_in_text(text: str, letter: str) -> int:
-    """Count occurrences of a letter in a text"""
-    return text.count(letter)
+@mcp.prompt()
+def debug_help(error_message: str, code: str = "") -> str:
+    """Get help debugging an error"""
+    prompt = f"Help me debug this error: {error_message}"
+    if code:
+        prompt += f"\n\nRelated code:\n```python\n{code}\n```"
+    prompt += "\n\nPlease explain what might be causing this error and suggest solutions."
+    return prompt
+
+
+@mcp.tool(
+    description="Add a list of numbers and return structured result with metadata",
```

Comment:
This says it returns structured result, but it just looks like a string?

Reply:
yeah, I misused 'structured' when I meant formatted but implemented structured results in latest commit.